spring-batch-core/src/main/java/org/springframework/batch/core/converter/DateToStringConverter.java

```diff
@@ -27,7 +27,11 @@
  *
  * @author Mahmoud Ben Hassine
  * @since 5.0.1
+ * @deprecated since 6.1 in favor of
+ * {@link ConversionServiceFactory#createConversionService()}. Scheduled for removal in
+ * 6.3 or later.
```

Comment:
Since all existing converters are public, I proposed deprecating them for removal in 6.3. Let me know if I should make any adjustments to it.